src/types/utils/isLoadingOnyxValue.ts

```diff
@@ -0,0 +1,7 @@
+import type {OnyxKey, UseOnyxResult} from 'react-native-onyx';
+
+function isLoadingOnyxValue(...results: Array<UseOnyxResult<OnyxKey, unknown>[1]>): boolean {
```

Comment:
Let's improve this type here, I think we should export `ResultMetadata` from onyx

Reply:
Will raise a PR in Onyx!

Reply:
Can you link the PR for this? @fabioh8010

Reply:
Sorry, forgot about it! Here is it: https://github.com/Expensify/react-native-onyx/pull/527

But I think we shouldn't hold this PR because of this, otherwise we will require another bump of multiple versions that would take more time.